Formula/cockroach.rb

```diff
@@ -19,6 +19,17 @@ class Cockroach < Formula
   depends_on "make" => :build
   depends_on "xz" => :build
 
+  # Compiling CockroachDB v19.1 with Go 1.12 changes the behavior of setrlimit
+  # in a way that causes CockroachDB to crash upon startup if kern.maxfiles is
+  # too low. This patch backports the upstream fix from
+  # cockroachdb/cockroach#37705, which will be included in the next release.
+  # Note that the pull request patch cannot be used directly, as the paths in
+  # the release tarball do not exactly match the paths in the Git repository.
+  patch do
+    url "https://gist.githubusercontent.com/benesch/4816feaf97e1ca35e2581d9288d9bdc9/raw/0b2d48195b43b29be573761e5895fba228f0f516/37705.patch"
```

Comment:
```suggestion
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/cockroach/v19.1.1-kern-maxfiles-w-go-1.12.patch"
```

Reply:
@benesch: Can you please make this change or give me permissions to push to your repo's branch?

Reply:
I think you should have permission?

![image](https://user-images.githubusercontent.com/882976/58212481-eccc0400-7cbc-11e9-93b7-bb2715f8fb65.png)

Regardless, I can make the change!

Reply:
Very strange. It was telling me I didn't have permissions.

Thanks!